Applications/Android/NNDetector/app/src/main/jni/nntrainer

```diff
@@ -0,0 +1 @@
+../../../../../../../builddir/android_build_result
```

Comment:
I think this is a symbolic link created by preapre_android_dep.sh. 
you can delete it.

Reply:
to pass "TAOS/pr-postbuild-build-android" check, the symbolic link is needed.

Reply:
yes. What I mean it will be generated after the .sh run. So we don't need to create it in source code level.